simply/actor.py

```diff
@@ -138,12 +139,25 @@ def create_prediction(self):
             self.pred["schedule"] = self.pred["pv"] - self.pred["load"]
 
     def update(self):
+        """Changes actor attributes according to the events in the current time step
+
+        The events can be the impact of schedule and trading on the battery soc, the bank / cost for
+        the actor in this time step, and an updated prediction which moves one time step ahead."""
```

Comment:
... update prediction horizon to current time step

note: time step already moved ahead before calling update()

Reply:
Refactored function for better readability